examples/docker/shiny-core/app-core.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import pandas as pd
+import plotly.express as px
+from shinywidgets import output_widget, render_widget
+from faicons import icon_svg
+
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+from shiny import App, Inputs, Outputs, Session, reactive, render, ui
+
+THRESHOLD_MID = 0.9
+THRESHOLD_MID_COLOR = "rgb(0, 137, 26)"
+THRESHOLD_LOW = 0.5
+THRESHOLD_LOW_COLOR = "rgb(193, 0, 0)"
+
+URI = "CONNECTION_STRING"
```

Comment:
read this from and env variable

Reply:
[fixed](https://github.com/ploomber/doc/pull/105/files#diff-c99010220192b232032cc417d9e26e799aa65a2dd2fbb5dd79b5595548293670R18-R19)